repos/system_upgrade/common/models/grubconfigerror.py

```diff
@@ -1,8 +1,8 @@
-from leapp.models import Model, fields
+from leapp.models import fields, Model
 from leapp.topics import SystemFactsTopic
 
 
 class GrubConfigError(Model):
     topic = SystemFactsTopic
 
-    error_detected = fields.Boolean(default=False)
```

Comment:
We cannot do this. Models are part of our stable API and they undergo deprecation process. We need to keep the original field, including its semantics, so if an user relied on this model, he will still find its original fields intact. I know that I have commented on the strange nature of this model and its meaning in our IRC communication, however, I did not mean to imply that you should fix the illogical design. Lets keep the original field intact, and still populate with True, whenever an error was encountered, so we do not break our API for other users.

That being said, I really, really like the pragmatic approach you have shown here - its a problem, so why not solve it right away since I am touching the code anyway.

Reply:
Thanks, for the explanation. I didn't realize this although it makes sense.